src/metax_api/tests/api/rest/base/views/common/write.py

```diff
@@ -79,6 +79,37 @@ def test_certain_create_fields_are_read_only_after_create(self):
         self.assertEqual(orig_user_created, response.data.get('user_created', None))
         self.assertEqual(orig_service_created, response.data.get('service_created', None))
 
+    def test_deletion_sets_removed_true_and_sets_value_for_date_removed(self):
+        response = self.client.post('/rest/datasets', self.test_new_data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
+
+        cr_id = response.data['id']
+        response = self.client.delete('/rest/datasets/%d' % cr_id)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT, response.data)
+
+        # Verify date_removed got set
+        response = self.client.get('/rest/datasets/%d?removed' % cr_id)
+        self.assertTrue(response.data['removed'] is True)
+        self.assertTrue(response.data.get('date_removed', False) and
+                        response.data.get('date_removed', '').startswith('2'))
```

Comment:
Nitpicking, but: This statement: response.data.get('date_removed', False) is redundant I think